### PR TITLE
feat(mobile): include partner's photos on map

### PR DESCRIPTION
**Pull request by BlackBadPinguin**
**Changes made in the PR**

- Adds a new option on the map page to include the Partner Images.

image

### CODE REVIEW

File: mobile/lib/modules/map/providers/map_marker.provider.dart [hunk 1]
```diff
@@ -26,9 +27,14 @@ Future<List<MapMarker>> mapMarkers(MapMarkersRef ref) async {
...
   }
 
+  if (mapState.withPartners) {
+    iswithPartners = true;
+  }
+
   final markers = await service.getMapMarkers(
     isFavorite: isFavorite,
     withArchived: isIncludeArchived,
+    withPartners: iswithPartners,
```
Comment (alextran1502) [inline mobile/lib/modules/map/providers/map_marker.provider.dart:37]:
```suggestion
    withPartners: isWithPartners,
```

**Comment by alextran1502** [comment]
Hello, I don't see the logic where the partner's assets get fetched. Can you help me point it out? Please also fix the formatting using as noted in the failed tests

**Comment by BlackBadPinguin** [comment]
Hey, I updated the formatting of the variable and the function.

The logic for the fetch is in the asset_api.

image

**Comment by alextran1502** [comment]
Ah duh, sorry, my bad

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**